_layouts/kakapo-js.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+  {% include head.html %}
+  <body>
+    {% include github_ribbon.html %}
+    <section id="banner" class="js-banner">
+      <h1>
+      Kakapo.js
```

Comment:
The layout should not be specific, get variables from the front matter like title and description and name it more generically , for example this layout could be documentation.html